apps/mark-scan/frontend/src/pages/admin_screen.tsx

```diff
@@ -62,6 +58,18 @@ export function AdminScreen({
 }: AdminScreenProps): JSX.Element {
   const { election } = electionDefinition;
   const logOutMutation = logOut.useMutation();
+  const getPrecinctSelectionQuery = getPrecinctSelection.useQuery();
+  const precinctSelection = getPrecinctSelectionQuery.data?.ok();
```

Comment:
Looking at this code, `precinctSelection` can end up `undefined` for one of two reasons: either 1) data hasn't finished loading yet, or 2) a precinct selection has actually not yet been made.

If it's not important to distinguish between these two cases, then this code is fine! But if it is important, you'll often see code like this:

```ts
if (!getPrecinctSelectionQuery.isSuccess) {
  return null; // Basically a data-loading UI state
}
const precinctSelection =  getPrecinctSelectionQuery.data.ok(); // No `?.` necessary
```

https://github.com/votingworks/vxsuite/blob/47369cc56898fe3d6cff2971e853141b92071b58/apps/admin/frontend/src/components/live_check_button.tsx#L13-L16

Reply:
This is a great point, we should return a loading state. I'll update callers of this query. This should also solve the coverage problem above.

Reply:
Thanks for leaving the explanations for cases that couldn't be updated!